react/components/FacetCheckboxList.js

```diff
@@ -100,7 +110,8 @@ const FacetCheckboxList = ({
         <ShowMoreFilterButton
           quantity={quantity - FACETS_RENDER_THRESHOLD}
           truncated={truncated}
-          toggleTruncate={() => openTruncated(truncated => !truncated)}
+          // eslint-disable-next-line no-shadow
```

Comment:
Why does this comment is needed? You can change the name of the callback parameter.

Reply:
As you suggeSince it was a function already written in this way I preferred not to touch it and enter the comment to obviate the problem reported by eslint. In the last commit, as you suggested, I have eliminated the comment and I have changed the name of the parameter.